### PR TITLE
Website: Responsive pages linked to from /educate

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by tanyaparker** [comment]
cc @breville

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md [hunk 1]
```diff
@@ -11,85 +12,85 @@ We are building a growing library of educational videos available for re-use by
 
 ## Computer Science Principles
 
-<div style="float:left; padding:10px; width:49%">
+<div class="col-50" style="float: left; padding: 10px;">
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md:15]: Should we make some more sophisticated layout styles longer-term?

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md [hunk 2]
```diff
@@ -137,15 +138,15 @@ We are building a growing library of educational videos available for re-use by
...
 
 <% facebook = {:u=>'https://youtu.be/jsUN0NV5RfQ'} %>
 <% twitter = {:url=>'https://youtu.be/jsUN0NV5RfQ', :related=>'codeorg', :text=>'Chris Bosh explains REPEAT UNTIL @codeorg'} %>
 <%=view :display_video_thumbnail, id: "repeat_until", video_code: "jsUN0NV5RfQ", caption: "Chris Bosh explains REPEAT UNTIL", play_button: 'center', facebook: facebook, twitter: twitter, download_path: "//s3.amazonaws.com/videos.code.org/2015/social/bosh-repeat-general.mp4" %>
 
 </div>
 
-<div style="float:left; padding:10px; width:49%">
+<div class="col-50" style="float: left; padding: 10px;">
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md:149]: The number of times this gets used makes me wonder if it'd be good to add some more styles now :)

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/educate/resources/videos.md:15]: Yes, I'll add as todo for when we make templates, etc.

**Pull request merged**